src/Frontend/Document.php

```diff
@@ -173,7 +180,9 @@ protected function makeView(): View
      */
     protected function makeTitle(): string
     {
-        return ($this->title ? $this->title.' - ' : '').Arr::get($this->forumApiDocument, 'data.attributes.title');
+        $onHomePage = rtrim($this->request->getAttribute('originalUri')->getPath(), '/') === '';
```

Comment:
Does `originalUri` means Flarum in a subdomain will still show the title for its index page?

Reply:
Considering that in https://github.com/askvortsov1/core/blob/58244a131ebf07ed6085052ce112ec77ad114e8a/src/Foundation/InstalledApp.php#L58-L67, "/" maps to the base path, I'm assuming that's not a player with OriginalMessages as well? (since that's just calling getUri)

Reply:
Oh right, the originalUri will be the original before the router, but after the basepath. So it should be the same value for `->getUri()` and `->getAttribute('originalUri')` when under the `flarum.forum.handler` router ?

In this case why not simply use `getUri()` as most documents will be in the forum router anyway?

Or is it about the admin homepage? Do we want to handle that page like another homepage or like a different non-homepage?

Reply:
originalUri is more explicit, but then again, I'm not sure that the admin dashboard would ever be set as the homepage on any reasonable forum... getUrl would probably be cleaner tbh.